src/cozmo/annotate.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2016 Anki, Inc.
+# Copyright (c) 2016-2017 Anki, Inc.
```

Comment:
Yeah, I believe new files should be 2017 now, and as we modify 2016 files they should become 2016-2017. Will clean any stragglers up in a later PR, I'm led to believe this isn't that critical given copyright duration.